backend/awsvpc/awsvpc.go

```diff
@@ -103,6 +102,34 @@ func (m *AwsVpcBackend) Init(extIface *net.Interface, extIP net.IP) (*backend.Su
 	}
 	ec2c := ec2.New(auth, region)
 
+	if m.cfg.RouteTableID == "" {
+		log.Infof("routeTableID not passed as config parameter, attempting to detect")
+		routeTableID, err := m.DetectRouteTableID(instanceID, ec2c)
+		if err != nil {
+			return nil, err
+		}
+		log.Info("detected routeRouteTableID: ", routeTableID)
+
+		m.cfg.RouteTableID = routeTableID
+	}
+
+	filter := ec2.NewFilter()
+	filter.Add("route.destination-cidr-block", l.Subnet.String())
+	filter.Add("route.state", "active")
+
+	resp, err := ec2c.DescribeRouteTables([]string{m.cfg.RouteTableID}, filter)
+	if err != nil {
+		return nil, fmt.Errorf("error describing route tables: %v", err)
```

Comment:
Since we now know that special (wildcard) permissions are required for Describe methods, what do you think about not failing here but continuing with a warning if the error is Unauthorized? Just include in the warning some text on what permission is required.